src/sbmlutils/report/sbmlinfo.py

```diff
@@ -35,221 +49,207 @@ class SBMLDocumentInfo:
-        """Create MathML content for the item.
-
-        :param sbase: SBML object for which MathML content is to be generated
-        :param math_type: specifies which math rendering mode to use
-        :return: formatted MathML content for the item
-        """
-
-        math = None
-        if math_type == "cmathml":
-            math = SBMLDocumentInfo._astnode_to_mathml(astnode)
-        elif math_type == "pmathml":
-            cmathml = SBMLDocumentInfo._astnode_to_mathml(astnode)
-            math = mathml.cmathml_to_pmathml(cmathml)
-        elif math_type == "latex":
-            latex_str = mathml.astnode_to_latex(astnode, model=model)
-            math = latex_str
-
-        return math
-
-
-    def _create_assignment_map(self) -> Dict:
+    def _create_assignment_map(self, model: libsbml.Model) -> Dict:
         """Create dictionary of symbols:assignment for symbols in model.
 
         This allows to lookup assignments for a given variable.
 
         :return: assignment dictionary for model
         """
-        values = dict()
-
-        # initial assignments
-        initial_assignments = []
-        assignment: libsbml.InitialAssignment
-        for assignment in self.model.getListOfInitialAssignments():
-            info = self.sbase_dict(assignment)
-            info["sid"] = assignment.getSymbol() if assignment.isSetSymbol() else None
-            info["assignment"] = self._math(assignment.getMath(), self.model, self.math_render)
-            info["units"] = derived_units(assignment)
-
-            initial_assignments.append(info)
-
-        values["assignments"] = initial_assignments
+        assignments: Dict[str, Dict] = {}
+
+        initial_assignment: libsbml.InitialAssignment
+        for initial_assignment in model.getListOfInitialAssignments():
+            pk_symbol = (
+                initial_assignment.getSymbol()
+                if initial_assignment.isSetSymbol()
+                else None
+            )
+            pk = self._set_pk(initial_assignment)
+            if pk_symbol:
+                assignments[pk_symbol] = {
+                    "pk": pk,
+                    "type": "initialAssignment",
+                }
 
-        # rules
-        rules = []
         rule: libsbml.Rule
-        for rule in self.model.getListOfRules():
-            info = self.sbase_dict(rule)
-            info["sid"] = rule.getVariable() if rule.isSetVariable() else None
-            info["assignment"] = self._math(rule.getMath(), self.model, self.math_render)
-            info["units"] = derived_units(rule)
-
-            rules.append(info)
-
-        values["rules"] = rules
+        for rule in model.getListOfRules():
+            pk_symbol = rule.getVariable() if rule.isSetVariable() else None
+            pk = self._set_pk(rule)
+            if pk_symbol:
+                typecode = rule.getTypeCode()
+                if typecode == libsbml.SBML_ASSIGNMENT_RULE:
+                    sbml_type = "assignmentRule"
```

Comment:
For sake of uniformity, can we keep  the ```sbml_type``` as ```AssignmentRule``` and similarly for others, instead of camelcase?

Reply:
resolved.